pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -596,3 +602,9 @@ func setChaosResourceImage() {
 		engine.Instance.Spec.Components.Runner.Image = ChaosRunnerImage
 	}
 }
+
+func getChaosType() {
+	if engine.Instance.Spec.ChaosType == "" {
```

Comment:
Can we also handle the case string `infra` also, like some error being reported? Or abort execution?

Reply:
yes, agree.